poetry.lock

```diff
@@ -0,0 +1,1013 @@
+[[package]]
```

Comment:
How did you make this file?

What extra considerations should we take now with this new way for each release?

Reply:
This file is managed by [poetry](https://python-poetry.org/docs/basic-usage/#installing-without-poetrylock). This is the file poetry uses to resolve all the dependencies for the environment and pin them to a specific version.

The only thing to take into account in a new release would be bumping the version in the `pyproject.toml` file. However, I think you can even enable another plugin in poetry to enable this automatically from git tags. @Yajo knows this better :smile:

Reply:
Yes, you can use https://github.com/mtkennerly/poetry-dynamic-versioning/